app/controllers/api/v1/site_controller.rb

```diff
@@ -197,8 +197,7 @@ def download_data
             render json: {error: "File not found: #{params[:filename]}"}, status: 404
           end
         rescue RuntimeError => e
-          error_context = ErrorTracker.format_extra_context(@study, {params: params})
-          ErrorTracker.report_exception(e, current_api_user, error_context)
+          ErrorTracker.report_exception_with_context(e, current_api_user, @study, params.to_unsafe_hash)
```

Comment:
did you mean to leave the '.to_unsafe_hash' here, or can we get rid of it like we did the others?

Reply:
Nope - that's a typo!  Good catch.

Reply:
4c4f415